code/API_definitions/qos-profiles.yaml

```diff
@@ -19,7 +19,7 @@ info:
     # Further info and support
 
     (FAQs will be added in a later version of the documentation)
-  termsOfService: http://swagger.io/terms/
+  termsOfService: https://swagger.io/terms/
```

Comment:
This and contact are optional.  Rather than updating this to https in the PR, should we just remove this and the contact in a separate PR.